static/css/main.less

```diff
@@ -83,7 +86,7 @@
    ========================================================================== */
 
 // Combined CF components.
-@import (less) "../../vendor/cf-concat/cf.less";
+@import (less) "@{loc-lib}/capital-framework/capital-framework.less";
```

Comment:
This one is okay. This is in `main.less` so it'll suck `capital-framework.less` out of `/src/vendor/capital-framework/capital-framework.less` and include it in `main.css` for serving. 

Let me know if I'm missing something with that.